autoscale-dockerswarm-container/pom.xml

```diff
@@ -40,43 +40,52 @@
         <dependency>
             <groupId>com.github.cafapi</groupId>
             <artifactId>caf-api</artifactId>
+             <scope>runtime</scope>
```

Comment:
- `<scope>` not aligned
- To be honest I don't understand why `caf-api` is a direct dependency at all.  My feeling is that all that should be here is the `autoscale-core` and the various drop-in modules (i.e. choice of codec, choice of config etc.).  Give me a shout if you want to discuss this one further.

Reply:
Sure Dermot. We can discuss this.